ops/hookcmds/_storage.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2025 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import json
+from typing import cast
+
+from ._types import Storage
+from ._utils import run
+
+
+def storage_add(name: str, count: int = 1):
```

Comment:
It seems like `storage-add` can add multiple at once. Should this signature be the following?

```suggestion
def storage_add(counts: dict[str, int]):
```

Reply:
Great catch, thanks, I had completely missed that, even though I've read that doc page several times!

Reply:
Fixed.